.dockerignore

```diff
@@ -1,13 +1,9 @@
-.git
-node_modules
-npm-debug.log
-config
-coverage
-lib
-.env.*
-.gitignore
-.graphqlconfig
-.prettierrc
-Jenkinsfile
-jest.config.js
-tsconfig.jest.json
\ No newline at end of file
+## Ignore everything by default
+*
+
+## Allow specific files and folders
+!assets
+!data
+!package*.json
+!scripts
+!src
```

Comment:
I think we might have to include `custom.d.ts` and `tsconfig,json` here as well. Perhaps `custom.d.ts` should be in the `src/`.

Reply:
Good catch! I'll add 'em to the list.